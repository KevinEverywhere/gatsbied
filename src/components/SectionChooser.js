import React from 'react'
import ItemChooser from './ItemChooser'

const SectionChooser = () => (
  <nav>
    <ul>
      <ItemChooser item="Work" />
      <ItemChooser item="Creative" />
      <ItemChooser item="Contact" />
    </ul>
  </nav>
)

export default SectionChooser
