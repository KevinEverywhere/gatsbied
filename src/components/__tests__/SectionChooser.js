import React from "react"
import renderer from "react-test-renderer"
import SectionChooser from "../SectionChooser"

describe("SectionChooser", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SectionChooser />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
