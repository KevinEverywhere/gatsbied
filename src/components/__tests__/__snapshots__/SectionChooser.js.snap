// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionChooser renders correctly 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="work"
      >
        Work
      </a>
    </li>
    <li>
      <a
        href="creative"
      >
        Creative
      </a>
    </li>
    <li>
      <a
        href="contact"
      >
        Contact
      </a>
    </li>
  </ul>
</nav>
`;
